org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/contentmodel/participants/diagnostics/LSPErrorReporter.java

```diff
@@ -80,12 +82,19 @@ public String reportError(XMLLocator location, String domain, String key, Object
 				}
 			}
 			message = str.toString();
+			
 		}
-
+		
 		// Fill diagnostic
 		diagnostics.add(new Diagnostic(toLSPRange(location, key, arguments, document), message,
 				toLSPSeverity(severity), XML_DIAGNOSTIC_SOURCE, key));
 
+		XMLSyntaxErrorCode code = XMLSyntaxErrorCode.get(key);

```

Comment:
Please remove this code, you have done that because you cannot have list of ranges, is that? If it that please remove it and disable test. I will do a change to support list of ranges.